readthedocs/api/v3/tests/test_projects.py

```diff
@@ -88,6 +121,63 @@ def test_own_projects_detail(self):
             self._get_response_dict('projects-detail'),
         )
 
+    @override_settings(ALLOW_PRIVATE_REPOS=True)
+    def test_own_projects_detail_privacy_levels_enabled(self):
+        self.client.credentials(HTTP_AUTHORIZATION=f"Token {self.token.key}")
+        response = self.client.get(
+            reverse(
+                "projects-detail",
+                kwargs={
+                    "project_slug": self.project.slug,
+                },
+            ),
+            {
+                "expand": (
+                    "active_versions,"
+                    "active_versions.last_build,"
+                    "active_versions.last_build.config"
+                ),
+            },
+        )
+        self.assertEqual(response.status_code, 200)
+        expected = self._get_response_dict("projects-detail")
+        expected["privacy_level"] = "public"
+        expected["external_builds_privacy_level"] = "public"
+        expected["active_versions"][0]["privacy_level"] = "public"
```

Comment:
This change should be done in the default JSON, I guess. The other tests would fail if we don't add these new fields on the JSON file used in tests.